kobo/settings/base.py

```diff
@@ -58,7 +58,7 @@
 ENKETO_CSRF_COOKIE_NAME = env.str('ENKETO_CSRF_COOKIE_NAME', '__csrf')
 
 # Limit sessions to 1 week (the default is 2 weeks)
-SESSION_COOKIE_AGE = 604800
+SESSION_COOKIE_AGE = env.int('KPI_DJANGO_SESSION_LENGTH_HOURS', 168) * 60 * 60
```

Comment:
Let's be consistent with Django name. I propose we use the environment variable name `DJANGO_SESSION_COOKIE_AGE`. No need to use use the prefix `KPI` since it's shared between KPI and KoBoCAT.

Reply:
As discussed [here](https://github.com/kobotoolbox/kobo-install/pull/215#discussion_r1101607071), I would modify the PR title and description.